docs/src/getting-started/updating-linuxcnc.adoc

```diff
@@ -193,11 +193,11 @@ interpreter is, then this section does not affect you.
 A seldom-used feature of LinuxCNC is support for pluggable interpreters,
 controlled by the undocumented `[TASK]INTERPRETER` INI setting.
 
-Versions of LinuxCNC before 2.9.0 used to handle an incorrect
+Versions of LinuxCNC before 2.9.1 used to handle an incorrect
```

Comment:
I think this was fixed in 2.9.0 and there was no change to 2.9.1, so I would not increase the version number here.

Reply:
Seems like I rushed something. Thank you.